### PR TITLE
Oscap policy entity

### PR DESCRIPTION
**Pull request by tstrych**
Hello please review @abalakh or @oshtaier 
Here are the [test](https://github.com/SatelliteQE/robottelo/pull/6248) please look too,
#144

### CODE REVIEW

File: airgun/entities/oscappolicy.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    def details(self, entity_name):
+        """Read the content from corresponding SCAP Policy dashboard,
+            clicking on the Name of SCAP Policy shows the dashboard
+
+        :param entity_name:
+        :return: dictionary with values from SCAP Policy Details View
+        """
+        view = self.navigate_to(self, 'All')
+        view.search(entity_name)
+        view.table.row(name=entity_name)['Name'].widget.click()
```
Comment (abalakh) [inline airgun/entities/oscappolicy.py]: Pls implement these steps as part of 'Details' navigate step

File: airgun/entities/oscappolicy.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+    prerequisite = NavigateToSibling('All')
+
+    def step(self, *args, **kwargs):
+        self.parent.new.click()
+
+
+@navigator.register(OSCAPPolicyEntity, 'Edit')
+class EditSCAPPolicy(NavigateStep):
+    """Navigate to edit existing SCAP Policy page."""
```
Comment (abalakh) [inline airgun/entities/oscappolicy.py]: nitpick: missing `Args` section

File: airgun/entities/oscappolicy.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+    def step(self, *args, **kwargs):
+        entity_name = kwargs.get('entity_name')
+        self.parent.search(entity_name)
+        self.parent.table.row(name=entity_name)['Actions'].widget.fill('Edit')
+
+
+@navigator.register(OSCAPPolicyEntity, 'Details')
+class DetailsSCAPPolicy(NavigateStep):
+    """To get data from SCAPPolicyDetail view"""
```
Comment (abalakh) [inline airgun/entities/oscappolicy.py]: >nitpick: missing `Args` section

File: airgun/views/oscappolicy.py [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(
+            self.title, exception=False) is not None
+
+
+class SCAPPolicyCreateView(BaseLoggedInView):
+    breadcrumb = BreadCrumb()
+
+    def fill(self, values):
```
Comment (abalakh) [inline airgun/views/oscappolicy.py]: Please add short docstring describing what's going on

**Comment by tstrych** [comment]
@abalakh please review

**Pull request merged**